watertap/costing/unit_models/ion_exchange.py

```diff
@@ -336,9 +336,11 @@ def cost_ion_exchange(blk):
             to_units=blk.costing_package.base_currency,
         )
     )
+    blk.costing_package.add_cost_factor(blk, "TIC")
```

Comment:
```suggestion
    blk.costing_package.add_cost_factor(blk, None)
```
I suppose we could also just use the local IX ``total_installed_cost_factor`` doing something like this? I am fine with the system TIC also and would let you make the decision.

Reply:
Okay, thanks for catching this. I will use the system TIC and modify IX costing model appropriately.